includes/rest-api/class-sensei-rest-api-home-controller.php

```diff
@@ -0,0 +1,255 @@
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Groups & Cohorts',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Quizzes',
+							'url'   => 'http://...',
+						],
+						[
+							'title' => 'Integrations',
+							'url'   => 'http://...',
+						],
+					],
+				],
+			],
+			'help'                  => [
+				// TODO: Replace with real implementation.
+				[
+					'title' => 'Get the most out of Sensei',
+					'items' => [
+						[
+							'title' => 'Sensei Documentation',
+							'url'   => 'http://...',
+							'icon'  => null,
+						],
+						[
+							'title' => 'Support forums',
+							'url'   => 'http://...',
+							'icon'  => null,
+						],
+						[
+							'title'        => 'Create a support ticket',
+							'url'          => null,
+							'extra_action' => [
+								'title' => 'Upgrade to Sensei Pro',
+								'url'   => 'https://...',
+							],
+							'icon'         => 'lock',
+						],
+					],
+				],
+			],
+			'guides'                => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'How to Sell Online Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Creating Video Courses',
+						'url'   => 'http://...',
+					],
+					[
+						'title' => 'How to Choose the Right Hosting Provider',
+						'url'   => 'http://...',
+					],
+				],
+				'more_url' => 'http://senseilms.com/category/guides/',
+			],
+			'news'                  => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro|interactive-blocks}.json.
+				'items'    => [
+					[
+						'title' => 'Introducing Interactive Videos For WordPress',
+						'date'  => '2022-08-31', // Localized for user.
+						'url'   => 'http://senseilms.com/inroducing-interactive-videos/',
+					],
+					[
+						'title' => 'New Block Visibility, Scheduled Content, and Group Features',
+						'date'  => '2022-08-09', // Localized for user.
+						'url'   => 'http://senseilms.com/conditional-content/',
+					],
+				],
+				'more_url' => 'https://senseilms.com/blog/',
+			],
+			'extensions'            => [
+				// TODO: Load from https://senseilms.com/wp-json/senseilms-home/1.0/{sensei-lms|sensei-pro}.json.
+				[
+					'title'        => 'Sensei LMS Post to Course Creator',
+					'image'        => 'http://senseilms.com/wp-content/uploads/2022/02/sensei-post-to-course-80x80.png',
+					'description'  => 'Turn your blog posts into online courses.',
+					'price'        => 0,
+					'product_slug' => 'sensei-post-to-course', // To be used with the installation function `Sensei_Setup_Wizard::install_extensions`.
+					'more_url'     => 'http://senseilms.com/product/sensei-lms-post-to-course-creator/',
+				],
+			],
+			'show_sensei_pro_promo' => false, // Whether we should show the promotional banner for Sensei Pro or not..
```

Comment:
👀  @Automattic/evergreen

I thought about returning text/images from the backend for the promotion but it looks too cumbersome. I considered also returning som kind of `active_promotion` attribute that we can populate with different keys (that frontend must know about).
But finally decided to go the simplest way: return a boolean specific for this case and we can iterate this API to do something more generic if we have other use cases in the future.

WDYT?

Reply:
I think just having a boolean is fine. Everything else can be kept pretty statically in the component we use for the Sensei Pro promo.

Reply:
While it could be cool to have different "ads", I feel like this isn't needed at the first moment. Maybe it's worth asking Ronnie/Andrei and implementing it as a low-priority task if needed, though.